docs/ko/reference/array/zipObject.md

```diff
@@ -0,0 +1,39 @@
+# zipObject
+
+두 개의 배열을 속성 이름과 해당 값의 객체로 결합해요.
+
+이 함수는 속성 이름이 포함된 배열과 해당 값이 포함된 배열을 입력받아, 속성 이름 배열의 요소를 키로 하고 값 배열의 요소를 값으로 하는 새로운 객체를 반환해요. 속성 이름 배열이 값 배열보다 길 경우, 남은 속성 이름의 값은 `undefined`로 채워져요.
```

Comment:
```suggestion
두 배열을 하나의 객체로 결합해요. 첫 번째 배열은 프로퍼티 이름을 나타내고, 두 번째 배열은 값을 나타내요. 

프로퍼티 이름을 나타내는 배열이 값을 나타내는 배열보다 길면, 값들은 `undefined`로 채워져요.
```

What about writing the docs a little simpler?

Reply:
@raon0211  Sounds like a great idea, shall I modify it and raise an additional pull request ?